components/Navbar/Navbar.tsx

```diff
@@ -40,21 +40,20 @@ export const Navbar = ({ links, ...props }: NavbarProps) => {
 
 export const NavbarItem = ({ children, route, ...props }: NavbarLinkProps) => {
   const pathname = usePathname();
-  const currentClasses = "border-b-orange text-orange font-bold";
-  const notCurrentClasses = "border-transparent text-blue font-medium";
+  const currentClasses = "border-b-orange text-orange";
+  const notCurrentClasses = "border-transparent text-blue";
   const isCurrent = pathname === route;
   const conditionalClasses = isCurrent ? currentClasses : notCurrentClasses;
 
-  // TODO: #385 Adjust styles of the navigation bar
   return (
     <li
-      className={`flex h-[60px] items-center border-b-[5px] pb-2 font-outfit text-base  ${conditionalClasses}`}
+      className={`flex h-[60px] items-center rounded-none border-b-[5px] pb-2 font-outfit text-base font-medium ${conditionalClasses}`}
       key={route}
       {...props}
     >
       <Link
         data-text={children}
-        className="duration-300 before:invisible before:block before:h-0 before:overflow-hidden before:font-extrabold before:content-[attr(data-text)] hover:font-extrabold hover:text-orange"
+        className="link block overflow-hidden font-medium duration-300 hover:text-orange"
```

Comment:
@alexxuss1996 Let's remove the animation, and we are ready to go!

```suggestion
        className="link block overflow-hidden font-medium hover:text-orange"
```